site/pages/pycon-thailand.en.rst

```diff
@@ -94,12 +94,25 @@ Sponsors
 
 .. container:: jumbotron clearfix
 
-   Our Sponsorship prospectus will be available soon.
```

Comment:
I think put the download link on the sponsorship page. since we want them to fill in the form.